log-classifier.cabal

```diff
@@ -51,6 +52,7 @@ library
                      , optparse-applicative
                      , reflection
                      , regex-tdfa
+                     , safe-exceptions
```

Comment:
You really like this `safe-exceptions` library, don't you :smile: 
Fine, include it then.